packages/devextreme/js/__internal/ui/m_lookup.ts

```diff
@@ -588,13 +588,25 @@ const Lookup = DropDownList.inherit({
       contentReady: this._contentReadyHandler.bind(this),
     });
 
+    this._popupFocusOutHandler($overlayContent);
+
     if (this.option('_scrollToSelectedItemEnabled')) this._popup._$arrow.remove();
 
     this._setPopupContentId(this._popup.$content());
 
     this._contentReadyHandler();
   },
 
+  _popupFocusOutHandler($overlayContent) {
+    $overlayContent.get(0).addEventListener('focusout', (e) => {
```

Comment:
Please, use eventsEngine syntax, not addEventListener.

Reply:
Other than that, you may notice that the focus from the content has moved to the address bar, the popup has closed, but the dx-lookup element still has a focused state:

<img width="1097" alt="image" src="https://github.com/user-attachments/assets/94b7e194-cd6a-40db-82ab-6d827d405c09">